numerics/src/FrictionContact/fclib_interface.c

```diff
@@ -250,6 +252,65 @@ int frictionContact_fclib_write(FrictionContactProblem* problem, char * title, c
 
   return info;
 
+}
+int frictionContact_fclib_write(FrictionContactProblem* problem, char * title, char * description, char * mathInfo,
+                                const char *path, int ndof)
+{
+  int info = 0;
+
+  fclib_local   *fclib_problem;
+
+  fclib_problem = (fclib_local*)malloc(sizeof(fclib_local));
+
+  fclib_problem->spacedim = problem->dimension;
+  fclib_problem->mu =  problem->mu;
+  fclib_problem->q =  problem->q;
+
+  fclib_problem->s =  NULL;
+
+  fclib_problem->info = (struct fclib_info*)malloc(sizeof(struct fclib_info)) ;
+  fclib_problem->info->title = title;
+  fclib_problem->info->description = description;
+  fclib_problem->info->math_info = mathInfo;
+
+  fclib_problem->W = (struct fclib_matrix*)malloc(sizeof(struct fclib_matrix));
+  fclib_problem->R = NULL;
+  fclib_problem->V = NULL;
+
+  fclib_problem->W->m = problem->M->size0;
+  fclib_problem->W->n = problem->M->size1;
+
+  CSparseMatrix * spmat = NM_triplet(problem->M);
+
+  fclib_problem->W = (struct fclib_matrix*)malloc(sizeof(struct fclib_matrix));
```

Comment:
fclib_problem->W is allocated a second time here (first time at line 276)

Reply:
Thank you. An ugly copy-paste.